scripts/VCSA/uninstall.sh

```diff
@@ -27,7 +27,7 @@ read_vc_information () {
                 VCENTER_ADMIN_PASSWORD=$OPTARG
                 ;;
             *)
-                echo Usage: $0 [-i vc_ip] [-u vc admin_username] [-p vc_admin_password] >&2
+                echo "Usage: $0 [-i vc_ip] [-u vc admin_username] [-p vc_admin_password]" >&2
```

Comment:
`common.sh` file so this isn't duplicated between install/uninstall. Or too much effort vs fixing the plugin install properly?